portfolio/src/main/java/com/google/sps/servlets/data/ServerInfo.java

```diff
@@ -0,0 +1,23 @@
+package com.google.sps.servlets.data;
+
+import java.util.Date;
+
+/** Class containing server statistics. */
+public final class ServerInfo {
```

Comment:
What's this class doing?

Reply:
I'm asking myself the same right now HAHAHA, let me fix that